src/style/index.css

```
@@ -400,6 +404,47 @@ button:disabled,
     }
 }
 
+.left-border-flash {
+    animation: borderPulse 1s infinite ease-out;
+    border-left: 10px solid #ffffff;
+    border-radius: 20px;
+}
+
+.right-border-flash {
+    animation: borderPulse 1s infinite ease-out;
+    border-right: 10px solid #ffffff;
+    border-radius: 20px;
+}
+
+.top-border-flash {
+    animation: borderPulse 1s infinite ease-out;
+    border-top: 10px solid #ffffff;
+    border-radius: 20px;
+}
+
+.bottom-border-flash {
+    animation: borderPulse 1s infinite ease-out;
+    border-bottom: 10px solid #ffffff;
+    border-radius: 20px;
+}
+
+
+@keyframes borderPulse {
```

just a minor suggestion, instead of camel case, use snake case with hyphen

```suggestion
@keyframes border-pulse {
```

also change it in the places this is used